src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -1553,23 +1554,26 @@ class ConsentGroupReview extends Component {
           button({
             className: "btn buttonPrimary floatRight",
             onClick: this.handleApproveInfoDialog,
-            isRendered: this.state.formData.consentForm.approvalStatus !== 'Approved' && this.state.isAdmin && this.state.readOnly === true,
+            isRendered: this.state.current.consentExtraProps.projectReviewApproved !== 'true' && this.state.isAdmin && this.state.readOnly === true,
             disabled: this.state.disableApproveButton
           }, ["Approve"]),
 
           /*visible for Admin in readOnly mode and if there are changes to review*/
           button({
             className: "btn buttonPrimary floatRight",
             onClick: this.handleApproveDialog,
-            isRendered: this.state.isAdmin && this.state.reviewSuggestion === true && this.state.formData.consentForm.approvalStatus === 'Approved' && this.state.readOnly === true
+            isRendered: this.state.isAdmin
+              && this.state.reviewSuggestion === true
+              && this.state.current.consentExtraProps.projectReviewApproved === 'true'
+              && this.state.readOnly === true
```

Comment:
Thanks for this, I like the broken out conditionals much better than single line ones.